cli_reference/index.adoc

```diff
@@ -0,0 +1,39 @@
+include::modules/common-attributes.adoc[]
+:context: cli-tools-overview
+
+toc::[]
+
+A user performs different tasks while working on {product-title} which includes a range of operations, for example:
+
+* Managing clusters
+* Building, deploying and managing applications
+* Managing deployment processes
+* Developing operators
+* Creating and maintaining catalogs
+
+{product-title} offers a set of command-line interface (CLI) tools that simplify these tasks by enabling users to perform various administration and development operations from the terminal.
+These tools expose simple commands to manage the applications, as well as the lower level tools to interact with each component of the system.
+
+[id="cli-tools-list"]
+== List of CLI tools
+//Some of these CLI tools are alternatively available in web console
+
+// Provide Getting started xref for all the CLI tools
+The following set of CLI tools are available in {product-title}:
+
+* xref:../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[OpenShift CLI (oc)]: This is the most commonly used CLI tool by {product-title} users. It helps both cluster administrators and developers to perform end-to-end operations across {product-title} using the terminal. Unlike web console, it allows the user to work directly with the project source code using command scripts.
+
+* xref:../cli_reference/developer_cli_odo/understanding-odo.adoc#understanding-odo[Developer CLI (odo)] (not in 4.8 and 4.9): The developers who primarily create and maintain applications on {product-title} do not require any detailed knowledge of {product-title}. `odo` helps the developers to to write, build, and debug applications on a cluster from the terminal without the need to have an understanding of cluster administration.
+
+* Helm CLI (not in 4.9): Helm is a package manager for Kubernetes applications which enables defining, installing, and upgrading applications packaged as Helm Charts. Helm CLI helps the user in deployment of applications and services to OpenShift Container Platform clusters using simple commands from the terminal.
+
+* xref:../cli_reference/kn-cli-tools.adoc#kn-cli-tools[Knative CLI (kn)]: `kn` helps {product-title} users, who develop and manage serv
+erless applications by installing OpenShift Serverless, to have simple interactions with Knative components on {product-title}. This enables the users to perform serverless computing tasks simple and concise.
+
+* xref:../cli_reference/tkn_cli/installing-tkn.adoc#installing-tkn[Pipelines CLI (tkn)]:  {product-title} uses OpenShift Pipelines as the continuous integration and continuous delivery (CI/CD) solution which internally uses Tekton. For the users of OpenShift Pipelines, `tkn` provides simple and intuitive commands to interact with OpenShift Pipelines using the terminal.
+
+* xref:../cli_reference/opm-cli.adoc#opm-cli[opm CLI]: `opm` helps the Operator developer to create and maintain the catalogs of Operators from the terminal.
```

Comment:
s/the Operator developer/Operator developers and cluster administrators/

Reply:
done